docs/en_US/SupportedFramework_Library.md

```diff
@@ -0,0 +1,51 @@
+# Framework and Library Supported 
+NNI provides Python API, supporting all frameworks and libraries written in python (`version >= 3.5`), and we have implemented many related examples and detailed their principles and running steps.
+
+
+
+## Support Framework:
+
+NNI supports all frameworks, as long as they are written in python. Some common frameworks are listed below, including NNI example tutorials based on them.
+
+* [PyTorch]https://github.com/pytorch/pytorch
+    <ul> 
+      <li><a href="../../examples/trials/mnist-distributed-pytorch">MNIST-pytorch</a><br/></li>
+      <li><a href="TrialExample/Cifar10Examples.md">CIFAR-10</a><br/></li>
+      <li><a href="../../examples/trials/kaggle-tgs-salt/README.md">TGS salt identification chanllenge</a><br/></li>
+      <li><a href="../../examples/trials/network_morphism/README.md">Network_morphism</a><br/></li>
+    </ul>
+* [TensorFlow]https://github.com/tensorflow/tensorflow
+    <ul> 
+      <li><a href="../../examples/trials/mnist-distributed">MNIST-tensorflow</a><br/></li>
+       <li><a href="../../examples/trials/ga_squad/README.md">Squad</a><br/></li>
+    </ul>
+* [Keras]https://github.com/keras-team/keras
+    <ul>
+      <li><a href="../../examples/trials/mnist-keras">MNIST-keras</a><br/></li>
+      <li><a href="../../examples/trials/network_morphism/README.md">Network_morphism</a><br/></li>
+    </ul>
+* [MXNet]https://github.com/apache/incubator-mxnet
+* [Caffe2]https://github.com/BVLC/caffe
+* [CNTK (Python language)]https://github.com/microsoft/CNTK
+* [Spark MLlib]http://spark.apache.org/mllib/
+* [Chainer]https://chainer.org/
+* [Theano]https://pypi.org/project/Theano/ <br/>
+
+In addition to these, other frameworks written in python are also supported by NNI.
```

Comment:
You are encouraged to contribute more examples for other NNI users. //add the how-to contribute link.

Reply:
fixed